hir/src/asm/isa.rs

```diff
@@ -1404,6 +1407,18 @@ impl MasmOp {
             Instruction::Debug(DebugOptions::LocalInterval(start, end)) => {
                 Self::DebugFrameRange(unwrap_u16!(start), unwrap_u16!(end))
             }
+            Instruction::Nop => Self::Nop,
```

Comment:
I'm fine with `Nop` being at the end, since it is new, but lets group the other ops together with their non-immediate versions so they are easy to find.

Reply:
Done in https://github.com/0xPolygonMiden/compiler/pull/221/commits/13af81c24e7ed03175fd59cd52bc64c1d2c10631